WalletWasabi/Wallets/Wallet.cs

```diff
@@ -464,4 +486,19 @@ public static Wallet CreateAndRegisterServices(Network network, BitcoinStore bit
 		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider);
 		return wallet;
 	}
+
+	public string Identifier => WalletName;
+	public bool CoinjoinEnabled => !IsUnderPlebStop;
```

Comment:
There is an option in the settings where you can enable or disable CoinJoins - it can be confusing. Also, you have IsMixable below and the difference is not significant according to the naming however it is very different if we look at the code. How about keeping the IsUnderPlebStop? or IsBalanceUnderMixTheshold.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/8754/commits/8e3bb1b5d5c00ded5763dc9c83fd281c235c0db2